core/dbt/adapters/base/relation.py

```diff
@@ -89,7 +89,10 @@ def matches(
             if not self._is_exactish_match(k, v):
                 exact_match = False
 
-            if self.path.get_lowered_part(k) != v.lower():
+            if (
+                self.path.get_lowered_part(k).strip(self.quote_character) != 
```

Comment:
I believe the flake8 error says:
```
core/dbt/adapters/base/relation.py:93:77: W291 trailing whitespace
```
which just looks like:
```suggestion
                self.path.get_lowered_part(k).strip(self.quote_character) !=
```